src/message/html/HtmlInAppLinks.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
```

Comment:
Continuing from the previous comment, the Link itself will not be a different component. It will just pass itself to the onPress event and the parent will have this very code.